model/task_queue_service_dependency.go

```diff
@@ -565,7 +566,12 @@ func checkUnmarkedBlockingTasks(t *task.Task, dependencyCaches map[string]task.T
 	blockingDeactivatedTasks, err := t.BlockedOnDeactivatedDependency(dependencyCaches)
 	catcher.Add(errors.Wrap(err, "can't get blocked status"))
 	if err == nil && len(blockingDeactivatedTasks) > 0 {
-		catcher.Add(errors.Wrapf(task.DeactivateDependencies(blockingDeactivatedTasks, evergreen.DefaultTaskActivator+".dispatcher"), "can't deactivate task '%s'", t.Id))
+		var deactivatedDependencies []task.Task
+		deactivatedDependencies, err = task.DeactivateDependencies(blockingDeactivatedTasks, evergreen.DefaultTaskActivator+".dispatcher")
+		catcher.Add(err)
+		if err != nil {
```

Comment:
Should this be err == nil, as in there was no problem deactivating the dependencies?

Reply:
yes.
Fixed